Rakefile

```diff
@@ -201,6 +201,8 @@ namespace :test do
   desc 'Run integration tests'
   RSpec::Core::RakeTask.new(:sdk) do |t|
     t.pattern = 'spec/integration/**/*.rb'
+    t.rspec_opts = '--color --format documentation --require spec_helper \
+    --format RspecJunitFormatter --out integration.results.xml'
```

Comment:
@vu-pham @chiendm can we add one more param --failure-exit-code=0, because we would like to trigger testwatch for unstable tests. By default rspec return exit code = 1 with failure tests